.github/workflows/stats.yaml

```
@@ -0,0 +1,17 @@
+on:
+  schedule:
+    # Run this once per day, towards the end of the day for keeping the most
+    # recent data point most meaningful (hours are interpreted in UTC).
+    - cron: "0 23 * * *"
+  workflow_dispatch: # Allow for running this manually.
+
+jobs:
+  j1:
+    name: Ignite CLI Repository Statistics
```

Maybe:

```suggestion
    name: Ignite Files Repository Statistics
```

I'm wondering if we need this feature though.